migrations/mysql/20210301140656_add_withdraws_table.sql

```diff
@@ -0,0 +1,28 @@
+-- +up
+-- +begin
+CREATE TABLE `withdraws`
+(
+    `gid`              BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+    `exchange`         VARCHAR(24)     NOT NULL DEFAULT '',
+
+    -- asset is the asset name (currency)
+    `asset`            VARCHAR(10)     NOT NULL,
+
+    `address`          VARCHAR(128)     NOT NULL,
+    `network`          VARCHAR(32)     NOT NULL DEFAULT '',
+
+    `amount`           DECIMAL(16, 8)  NOT NULL,
+    `txn_id`           VARCHAR(256)    NOT NULL,
+    `txn_fee`          DECIMAL(16, 8)  NOT NULL DEFAULT 0,
+    `txn_fee_currency` VARCHAR(32)     NOT NULL DEFAULT '',
+    `time`             DATETIME(3)     NOT NULL,
+
+    PRIMARY KEY (`gid`),
+    UNIQUE KEY `txn_id` (`exchange`, `txn_id`)
```

Comment:
Is the unique constraint name `txn_id` must be unique to all tables? If it is, we should append a prefix to it to prevent naming collision, such as `withdraw_txn_id`.

Reply:
the key name is table wide, so it's fine. ;-)

![image](https://user-images.githubusercontent.com/50894/111055335-d6092980-84af-11eb-927f-42bc481bd08c.png)